Userland/Libraries/LibWeb/CSS/StyleValues/ScrollbarGutterStyleValue.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024, the Ladybird developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/CSS/StyleValue.h>
+
+namespace Web::CSS {
+
+class ScrollbarGutterStyleValue final : public StyleValueWithDefaultOperators<ScrollbarGutterStyleValue> {
+public:
+    static ValueComparingNonnullRefPtr<ScrollbarGutterStyleValue> create(bool stable, bool both_edges)
+    {
+        return adopt_ref(*new (nothrow) ScrollbarGutterStyleValue(stable, both_edges));
+    }
+    virtual ~ScrollbarGutterStyleValue() override = default;
+
+    bool stable() const { return m_stable; }
+    bool both_edges() const { return m_both_edges; }
+
+    virtual String to_string() const override
+    {
+        if (m_both_edges) {
+            return "stable both-edges"_string;
+        }
+        if (m_stable) {
+            return "stable"_string;
+        }
+        return "auto"_string;
+    }
+
+    bool properties_equal(ScrollbarGutterStyleValue const& other) const { return m_stable == other.m_stable && m_both_edges == other.m_both_edges; }
+
+private:
+    ScrollbarGutterStyleValue(bool stable, bool both_edges)
```

Comment:
Should these use enum values (something other than magic bools)?

Reply:
Would a `ScrollbarGutterBehavior` enum with `Auto`, `Stable` and `BothEdges` for values work here?

Reply:
Yeah that's a good name for it. I'll update this on Saturday and fix up the test expectation

Reply:
I'm surprised the json file doesn't autogenerate enums for us 🤔

Reply:
It does, that's what enums.json is for. :^)

Should be able to use the generated enum-from-string function in parsing. Look at other parsing code for examples, or I can take a proper look later.

Reply:
I don't think enum from string can work because it's a two value property. It just so happens that the only case where both-edges is valid at parse is when the other value is stable. So it's effectively a tri-state. I've used the enum a bit more now rather than the bools